hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/rollback/ListingBasedRollbackHelper.java

```diff
@@ -116,6 +119,15 @@ public ListingBasedRollbackHelper(HoodieTableMetaClient metaClient, HoodieWriteC
                   .withDeletedFileResults(filesToDeletedStatus).build());
         }
         case APPEND_ROLLBACK_BLOCK: {
+          // collect all log files that is supposed to be deleted with this rollback
+          Map<FileStatus, Long> writtenLogFileSizeMap = Collections.EMPTY_MAP;
+          if (config.useFileListingMetadata()) {
+            Path partitionPath = FSUtils.getPartitionPath(config.getBasePath(), rollbackRequest.getPartitionPath());
+            writtenLogFileSizeMap = metaClient.getFs().exists(partitionPath) ? FSUtils.getAllLogFiles(metaClient.getFs(), partitionPath,
```

Comment:
what does `FSUtils.getAllLogFiles` return if the partition does not exist? can we avoid the .exists() call

Reply:
why can we get the logfiles only for that fileID here?  seems like you are fetching all log files in that partition?